Src/ST-LIB_LOW/Math/Math.cpp

```diff
@@ -1,68 +1,60 @@
 #include "ST-LIB_LOW/Math/Math.hpp"
 #include "CORDIC/CORDIC.hpp"
 
-int32_t Math::array[4] = {0,0,0,0};
-int32_t *Math::pointer1 = array;
-int32_t *Math::pointer2 = (int32_t *) array + 1;
-int32_t *Math::pointer3 = (int32_t *) array + 2;
-int32_t *Math::pointer4 = (int32_t *) array + 3;
+array<int32_t, 4> Math::pointers = {0,0,0,0};
```

Comment:
```suggestion
array<int32_t, 4> Math::pointers = { 0 };
```

Esto inicializa todo a cero. Es mejor porque es independiente del tamaño del array, si lo cambiaras no tienes que añadir o quitar ceros. Tambien es mas limpio